pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -28,8 +28,9 @@ import (
 )
 
 const (
-	maxQueueSize = 1 << 19 // 524288. ~500MB assuming 1KB per record
-	insertQuery  = `INSERT INTO flows (
+	DefaultMergeWithTtlTimeout = 14400
```

Comment:
s/Ttl/TTL

is this used anywhere, I couldn't find it

Reply:
I've deleted it. Currently, we don't change ttl related parameter in antctl